src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -68,7 +83,55 @@ public FilterPanel(UI ui, IModel model, PanelControl parentPanelControl, int pan
                 getStyleClass().remove("panel-focused");
             }
         });
-        
+        setupKeyboardShortcuts();
+    }
+
+    private void setupKeyboardShortcuts() {
+        addEventHandler(KeyEvent.KEY_PRESSED, event -> {
+            if (MAXIMIZE_WINDOW.match(event)) {
+                ui.maximizeWindow();
+            }
+            if (MINIMIZE_WINDOW.match(event)) {
+                ui.minimizeWindow();
+            }
+            if (DEFAULT_SIZE_WINDOW.match(event)) {
+                ui.setDefaultWidth();
+            }
+            if (SWITCH_DEFAULT_REPO.match(event)) {
```

Comment:
@rd1992 Sorry, you can ignore this. I am going to change my PR since the global shortcut is messing with the test. I will keep the switch repo shortcut in the panel.